SPECS/logrotate/logrotate.spec

```diff
@@ -64,28 +61,41 @@ install -p -m 644 examples/{b,w}tmp %{buildroot}%{_sysconfdir}/logrotate.d/
 %ghost %verify(not size md5 mtime) %attr(0644, root, root) %{_localstatedir}/lib/logrotate/logrotate.status
 
 %changelog
-*   Wed Jul 21 2021 Henry Beberman <henry.beberman@microsoft.com> - 3.18.1-1
--   Update to version 3.18.1
--   Add default logrotate systemd service and logrotate.conf
-*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> - 3.16.0-2
--   Added %%license line automatically
-*   Fri Apr 24 2020 Pawel Winogrodzki <pawelwi@microsoft.com> 3.16.0-1
--   Updated to 3.16.0.
--   License verified.
--   Updated 'Url' and 'Source0' tags.
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 3.14.0-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
-*   Wed Sep 05 2018 Srivatsa S. Bhat <srivatsa@csail.mit.edu> 3.14.0-1
--   Update to version 3.14.0
-*   Mon Jul 31 2017 Kumar Kaushik <kaushikk@vmware.com> 3.11.0-3
--   Creating /etc/logrotate.d folder as part of package installation, Bug#1878180.
-*   Wed Jun 14 2017 Anish Swaminathan <anishs@vmware.com> 3.11.0-2
--   Mark config files as noreplace
-*   Fri Apr 14 2017 Kumar Kaushik <kaushikk@vmware.com> 3.11.0-1
--   Updating version to 3.11.0
-*   Mon Mar 13 2017 Alexey Makhalov <amakhalov@vmware.com> 3.9.1-3
--   Compilation for gcc 6.3
-*   Tue May 24 2016 Priyesh Padmavilasom <ppadmavilasom@vmware.com> 3.9.1-2
--   GA - Bump release of all rpms
-*   Wed Jun 24 2015 Divya Thaluru <dthaluru@vmware.com> 3.9.1-1
--   Initial build. First version
+* Tue Apr 19 2022 Cameron Baird <cameronbaird@microsoft.com> - 3.18.1-2
+- Reenable systemd hardening configs ProtectClock, ProtectHostname, and ProtectKernelLogs
+
+* Wed Jul 21 2021 Henry Beberman <henry.beberman@microsoft.com> - 3.18.1-1
+- Update to version 3.18.1
+
```

Comment:
Nit: empty line inserted at the wrong place here (but thanks for cleaning this up!)